topiary/Cargo.toml

```diff
@@ -23,7 +23,6 @@ prettydiff = { workspace = true }
 regex = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
-toml = { workspace = true }
 tree-sitter-facade = { workspace = true }
 unescape = { workspace = true }
```

Comment:
I see that the tree-sitter-language dependencies are still required in the core engine. Maybe not the goal of this PR, but it would be great to be able to abstract the language as well so that it can be passed as a function argument without needing to build languages we do not need.

Reply:
That is absolutely the plan! Thanks for your feedback!